bin/dax_tools/GenerateProcessorTemplate

```diff
@@ -141,8 +142,8 @@ class Processor_{name}(ScanProcessor):
 
 SESSION_LEVEL_TEMPLATE = DEFAULT_TEMPLATE+"""
 # Format for the spider command line
-SPIDER_FORMAT = '''python {{spider}} -v --spm {{spm}} \
--p {{proj}} -s {{subj}} -e {{sess}} -d {{dir}} --suffix "{{suffix_proc}}"'''
+SPIDER_FORMAT = '''python {{spider}} -p {{proj}} -s {{subj}} -e {{sess}}\
```

Comment:
Yes for the space.

You can remove the pathsettings. I thought I remove all of them.

Ben

> On Oct 2, 2015, at 19:48, Brian D. Boyd <notifications@github.com> wrote:
> 
> In bin/dax_tools/GenerateProcessorTemplate:
> 
> > @@ -141,8 +142,8 @@ class Processor_{name}(ScanProcessor):
> >  
> >  SESSION_LEVEL_TEMPLATE = DEFAULT_TEMPLATE+"""
> >  # Format for the spider command line
> > -SPIDER_FORMAT = '''python {{spider}} -v --spm {{spm}} \
> > --p {{proj}} -s {{subj}} -e {{sess}} -d {{dir}} --suffix "{{suffix_proc}}"'''
> > +SPIDER_FORMAT = '''python {{spider}} -p {{proj}} -s {{subj}} -e {{sess}}\
> need a space before the \
> 
> —
> Reply to this email directly or view it on GitHub.
>